cmd/ciReleaseDeploy.go

```diff
@@ -95,6 +96,15 @@ var ciReleaseDeployCmd = &cobra.Command{
 		if len(deploymentTimeout) == 0 {
 			deploymentTimeout = "15m"
 		}
+		// Convert deploymentTimeout to integer of seconds
+		deploymentTimeoutSeconds := 900
+		deploymentTimeoutDuration, err := time.ParseDuration(deploymentTimeout)
+		if err != nil {
+		    deploymentTimeoutSeconds = 900
```

Comment:
Instead of assigning the value 900 again, check if there is an error and if yes, exit with an error message. Something like "deploymentTimeout is not a valid duration" or "could not convert bla bla bla".

Reply:
@Rade333 Set log notice. No need to exit if timeout is invalid. + Test scenarios are impossible in current state if we throw error.